src/components/OpeningTimes.js

```diff
@@ -34,12 +34,9 @@ function OpeningTimes({
         return (
             <span className="until">
                 <span className="open">
-                    Open now
+                    Hours may vary, see open times
                 </span>
                 {" "}
```

Comment:
Might be able to get rid of this element as well, as theres nothing after it now